src/core/dom/layout/size-observer.js

```diff
@@ -5,7 +5,11 @@ import {getWin} from '#core/window';
 
 import {LayoutSizeDef} from './rect';
 
-/** @typedef {!LayoutSizeDef|!ResizeObserverSize} TargetSize */
+/** @typedef {LayoutSizeDef|ResizeObserverSize} TargetSize */
+/**
+ * @template {TargetSize} Size
+ * @typedef {function(Size):void} SizeCallback
```

Comment:
nit: I don't know if you know this, but you can use TS Syntax in the blocks, i.e. `{(s:Size) => void}`

Reply:
I did not. Neat!